sql/core/src/main/scala/org/apache/spark/sql/execution/stat/StatFunctions.scala

```diff
@@ -91,7 +100,13 @@ object StatFunctions extends Logging {
     }
     val summaries = df.select(columns: _*).rdd.aggregate(emptySummaries)(apply, merge)
 
-    summaries.map { summary => probabilities.map(summary.query) }
+    summaries.map { summary =>
+      try {
+        probabilities.map(summary.query)
+      } catch {
+        case e: SparkException => Seq.empty[Double]
```

Comment:
Please do not use the Exception handling for this purpose. Instead, you can return None.